src/components/SubscriptionTable/__tests__/SubscriptionTable.test.tsx

```diff
@@ -21,102 +22,115 @@ describe('Subscription Table', () => {
     factories.subscription.build({
       number: '1234',
       contractNumber: '2345',
-      quantity: '1',
+      quantity: '',
       endDate: '2022-10-24T04:00:00.000Z',
       status: 'Active',
       startDate: '2021-10-24T04:00:00.000Z'
     })
   ]);
 
   it('renders correctly', async () => {
-    const { container } = render(<Table subscriptions={get('data')} />);
+    const { getAllByText } = render(<Table subscriptions={get('data')} />);
 
-    expect(container).toMatchSnapshot();
+    getAllByText('1234').forEach((el) => {
+      expect(el).toBeInTheDocument();
+    });
   });
 
   it('shows not available for missing values', () => {
-    def('data', () => [
-      factories.subscription.build({
-        number: '',
-        contractNumber: '',
-        quantity: '',
-        endDate: '',
-        status: '',
-        startDate: ''
-      })
-    ]);
-
-    const { container } = render(<Table subscriptions={get('data')} />);
+    render(<Table subscriptions={get('data')} />);
 
-    expect(container).toMatchSnapshot();
+    expect(screen.getByText('Not Available')).toBeInTheDocument();
```

Comment:
It might be better to specifically target quantity with this check to ensure the empty value is what is rendered as "Not Available"